src/utils/env/index.ts

```diff
@@ -1,8 +1,19 @@
-import { $TSFixMe } from '../../commands/types.js'
+import { EnvVar, EnvVarValue, ExtendedNetlifyAPI } from '../../commands/api-types.d.js'
+import type {
+  Context,
+  EnviromentVariables,
+  $TSFixMe,
+  Scope,
+  EnvironmentVariableSource,
+  EnvironmentVariableScope,
+} from '../../commands/types.js'
 import { error } from '../command-helpers.js'
+import { APIEnvError } from '../types.js'
 
-export const AVAILABLE_CONTEXTS = ['all', 'production', 'deploy-preview', 'branch-deploy', 'dev']
-export const AVAILABLE_SCOPES = ['builds', 'functions', 'runtime', 'post_processing']
+import { GetEnvelopeEnvParams, ProcessedEnvVars } from './types.js'
+
+export const AVAILABLE_CONTEXTS: Context[] = ['all', 'production', 'deploy-preview', 'branch-deploy', 'dev']
```

Comment:
Can we change that to `DeployContext`, context on its own is too vague

```suggestion
export const AVAILABLE_CONTEXTS: DeployContext[] = ['all', 'production', 'deploy-preview', 'branch-deploy', 'dev']
```